### PR TITLE
fix security issues and bugs

### PR DESCRIPTION
**Pull request by tiffanynwyeung**
## Changes
- update Pipfile dependencies to latest to address security issues with Django according to Github
- fix missing `participant`s in views for `ManageOZ` for `moderator.py` and `SignupPlayers` in `volunteer.py`
- remove `player_required` decorator from `PlayerList` view for `player.py`
- remove useless settings view to prevent people from routing there
- nit: filter signup invite count by distinct emails
- style nit: add commas on some lists

### CODE REVIEW

File: app/views/player.py [hunk 1]
```diff
@@ -136,7 +136,6 @@ def post(self, request):
 
 
 @method_decorator(running_game_required, name='dispatch')
-@method_decorator(player_required, name='dispatch')
```
Comment (tso) [inline app/views/player.py]: fix this and lgtm

File: app/urls.py [hunk 2]
```diff
@@ -21,7 +21,6 @@
 
     # Default dashboard pages
     path("dashboard", views.DashboardView.as_view(), name='dashboard'),
-    path("settings", views.SettingsView.as_view(), name='settings'),
```
Comment (tso) [inline app/urls.py]: dark mode when!?!?!

Comment (tiffanynwyeung) [reply app/urls.py]: when v2.0 pls

Comment (slnt) [reply app/urls.py]: DARK MODE WHEN

**Pull request merged**